src/query/functions/aggregation/base.go

```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/m3db/m3/src/query/parser"
 )
 
-type aggregationFn func(values []float64, indices []int) float64
+type aggregationFn func(values []float64, buckets []int) float64
```

Comment:
buckets is misleading ? since its  a single bucket

Reply:
Haha changed it everywhere except here :(